src/main_page/Main_page.js

```diff
@@ -138,7 +138,7 @@ class MainPage extends React.Component {
 
 
     async componentDidMount() {
-        /* let result = axios.get(`http://localhost:8041`)
+        /* let result = axios.get(`http://localhost:9041`)
              .then(res => {this.setState({citiesList: res.data})
```

Comment:
Do you need this commented code?

Reply:
this code is supposed to be used when BE Controller is done, so may it leave ok? 
Or what would you say?